src/main/java/hudson/plugins/sonar/SonarRunnerBuilder.java

```diff
@@ -258,23 +288,21 @@ private boolean performInternal(Run<?, ?> run, FilePath workspace, Launcher laun
     env.put("SONAR_RUNNER_OPTS", getJavaOpts());
 
     long startTime = System.currentTimeMillis();
-    int r;
+    int exitCode;
     try {
-      r = executeSonarRunner(run, workspace, launcher, listener, args, env);
+      exitCode = executeSonarRunner(run, workspace, launcher, listener, args, env);
```

Comment:
You can take the opportunity to rename the method: executeSonarQubeScanner() or executeSQScanner()